client/src/modules/authorization/components/SignInForm.jsx

```diff
@@ -159,7 +160,11 @@ class SignInForm extends PureComponent {
           <FormattedMessage id="authorization.auth-box.sign-in" />
         </h1>
         {signInErrorId && this.renderSignInError()}
-        <form className="sign-in__form" noValidate>
+        <form
+          className="sign-in__form"
+          noValidate
```

Comment:
what does this noValidate prop do?

Reply:
Maybe this prop has an unclear name. In some special situations, you can pass it to AuthInput to prevent displaying a green color even if the input is valid. This component is used in different forms so it can be configured a little bit different.

Reply:
maybe 
`noSuccessTheme`  would be better?

`noValidate` sounds like there is a validate function somewhere, and it should not validate when this prop is passed